docs/OESDLBugBar.md

```diff
@@ -0,0 +1,84 @@
+# Open Enclave SDL Bug Bar
+
+The information listed in this “bug bar” is intended to help Open Enclave SDK developers to triage bugs and determine the bug severity in terms of security. 
+
+This bug bar and the ratings are derived based of Microsoft SDL Bug bar to apply for the enclaves, specifically to the enclaves created using Intel SGX technology. It is highly recommended to fix all known bugs with critical, important, or moderate severity prior to release. 
+
+The bug bar describes different severities for the bugs affecting enclaves. Enclaves are the privileged execution environment which is a security domain. There is a security boundary between enclave and rest of the system, this is a stronger security boundary that should be defended. It is important to understand the enclave security boundary, it should be apt to consider this as a confidentiality & integrity boundary. Anything that is inside enclave should never be accessible or modified from outside of the enclave. 
+
+Enclave security boundary is quite interesting, everything that is outside this boundary is adversarial which can fully control the existence of the enclave but can’t access or modify the data over inside the enclave crossing the enclave security boundary. Things that are outside the enclave security boundary are the current application, other enclaves, OS kernel, VMM/Hypervisor & SMM. Except the CPU everything is untrusted for enclaves. 
+
+
+## Enclave – Vulnerability Type Pivot 
+-------------------------------------
+
+* ### Spoofing
+	**Critical**
+    * A malicious enclave spoofs the identity of a victim enclave
+  	
+    -------------
+
+* ### Tampering	
+    **Critical**
+    * Any modification of enclave image/data that compromise the integrity of the enclave
+
+            Examples:
+            •	Tampering the enclave image in plain text and keep the measurements same
+    
+    -------------
+
+* ### Repudiation
+    * N/A
+
+    -------------
+
+* ### Information Disclosure
+    **Critical**
+	* Disclosing enclave memory contents in the plain-text (not encrypted) 
+
+            Examples:
+            •	OCALLS returning uninitialized memory from enclave to outside	
```

Comment:
do you mean "passing" (as in a parameter) rather than returning?

Reply:
I agree, passing makes more sense.